client/src/pages/home/home.js

```diff
@@ -1,52 +1,50 @@
-import React,{useState} from 'react';
-import PropTypes from 'prop-types';
-import { withStyles } from '@material-ui/core/styles';
-import Paper from '@material-ui/core/Paper';
-import Grid from '@material-ui/core/Grid';
-import Form from '../../components/Form';
-import LMap from '../../components/leafletMap';
-import Results from '../../components/Results';
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import Form from "../../components/Form";
+import LMap from "../../components/leafletMap";
+import Results from "../../components/Results";
 
-
-const styles = theme => ({
+const styles = (theme) => ({
   root: {
     flexGrow: 1,
   },
   paper: {
     padding: theme.spacing(2),
-    textAlign: 'center',
+    textAlign: "center",
     color: theme.palette.text.secondary,
-    lineHeight: '45px',
-    boxShadow: 'none',
+    lineHeight: "45px",
+    boxShadow: "none",
   },
   intro: {
-    textAlign: 'center',
-    lineHeight: '45px',
-    fontSize: '25px',
+    textAlign: "center",
+    lineHeight: "45px",
+    fontSize: "25px",
   },
   map: {
-    display: 'flex',
-    justifyContent: 'center',
-    marginTop: '5rem',
-    marginBottom: '5rem'
+    display: "flex",
+    justifyContent: "center",
+    marginTop: "5rem",
+    marginBottom: "5rem",
   },
 });
 
 function Home(props) {
   const { classes } = props;
-  const [results, setResults] = useState([]); 
+  const [results, setResults] = useState([]);
 
   return (
     <div className={classes.root}>
       <Grid container>
         <Grid item xs={12}></Grid>
         <Grid item sm={12}>
-          <Paper className={classes.paper}>
-          </Paper>
+          <Paper className={classes.paper}></Paper>
         </Grid>
         <Grid item xs={12}>
           <Grid item xs={4}></Grid>
-          <Paper className={classes.map}>
+          <Paper elevation={0} className={classes.map}>
```

Comment:
What is elevation for?

Reply:
I had to override the default elevation in paper so that a border line at the bottom of the map went away